README.md

```diff
@@ -69,7 +69,7 @@ Below are the methods this SDK exposes. See [Iterable's API Docs](https://api.it
 API [(see required API payload here)](https://api.iterable.com/api/docs#In-app_getMessages):
 
 ```ts
-getInAppMessages: (payload: InAppMessagesRequestParams, showMessagesAutomatically?: boolean) => Promise<TrackConsumeData> | PaintInAppMessageData
+getInAppMessages: (payload: InAppMessagesRequestParams, showMessagesAutomatically?: boolean | { display: 'deferred' | 'immediate' }) => Promise<TrackConsumeData> | PaintInAppMessageData
```

Comment:
How do `true` and `false` behaviors differ from `{display: 'deferred'}` and `{display; 'immediate'}`? I.e., are there 4 different behaviors? Do you think it'd be worth providing a sentence in the readme to explain how they differ?

Reply:
yes I can explain! essentially, these two do the same thing:

```
getMessages(options, true);
getMessages(options, { display: 'immediate' })
```

The only reason I'm keeping the boolean value is so that we remain backwards-compatible and this next release doesn't break any implementations that the customers have already written. In other words, I don't want customers to start getting errors if they go from `v0.1.3` -> `v0.2.0` (which is what the next release is going to be).

My plan is to follow the semantic versioning strategy and only bump the major version when breaking changes are introduced. So `true | false` values will be deprecated once there is another major version bump, but until then, we'll keep it around.